src/diamond/handler/tsdb.py

```diff
@@ -1,370 +1,375 @@
 # coding=utf-8
-

```

Comment:
I was afraid you were going to say something like that.  Is it possible to break into 2 commits then so its easier to see changes? 1st commit is the newline change and 2nd commit is your actual code changes?